Set pullup on BTN_BACK pin

shitcreek opened the pull request:
```*this = matrix.vectors[0] * _x + matrix.vectors[1] * _y + matrix.vectors[2] * _z;```

- Bed leveling always returns 1 for Z `from_steppers` (M114 D)
- Z axis movement is locked

credit to @Evgeny-SPB
#15610

### Related Issues
#15562  #15495

shitcreek committed:
partially revert matrix rotation code - vector_3.cpp - AUTO_BED_LEVELING_3POINT

- Bed leveling always returns 1 for Z `from_steppers` (M114 D)
- Z axis movement is locked
credit to @Evgeny-SPB
#15610

Evgeny-SPB commented:
This PR addresses the same issue 
https://github.com/MarlinFirmware/Marlin/pull/15636

lukaszzamel commented:
Looks like it also fixes #15494

shitcreek committed:
Enable pull up for BACK BTN - ultralcd.cpp #15652

Back button did not work because internal pull up resistor was not enabled, leaving the pin floating when not depressed. This is missing in branches 1.1.x and 2.0

credit to @ellensp
#15652

thinkyhead committed:
Merge branch 'bugfix-2.0.x' into bugfix-2.0.x
